authors/apps/articles/models.py

```diff
@@ -129,3 +129,12 @@ class Favorite(models.Model):
         related_name='favorites')
     article_id = models.ForeignKey(
         Article, on_delete=models.CASCADE, related_name='favorites')
+
+
+class Bookmark(models.Model):
```

Comment:
Please return a string representation of this model, it will really help the rest of us when implementing other features